dot/rpc/subscription/listeners.go

```diff
@@ -410,13 +410,13 @@ func (l *RuntimeVersionListener) Listen() {
 		return
 	}
 	ver := modules.StateRuntimeVersionResponse{}
-	ver.SpecName = string(rtVersion.SpecName())
-	ver.ImplName = string(rtVersion.ImplName())
-	ver.AuthoringVersion = rtVersion.AuthoringVersion()
-	ver.SpecVersion = rtVersion.SpecVersion()
-	ver.ImplVersion = rtVersion.ImplVersion()
-	ver.TransactionVersion = rtVersion.TransactionVersion()
-	ver.Apis = modules.ConvertAPIs(rtVersion.APIItems())
+	ver.SpecName = string(rtVersion.SpecName)
+	ver.ImplName = string(rtVersion.ImplName)
+	ver.AuthoringVersion = rtVersion.AuthoringVersion
+	ver.SpecVersion = rtVersion.SpecVersion
+	ver.ImplVersion = rtVersion.ImplVersion
+	ver.TransactionVersion = rtVersion.TransactionVersion
+	ver.Apis = modules.ConvertAPIs(rtVersion.APIItems)
```

Comment:
can we move this to a helper function `newStateRuntimeVersionResponse(ver Version)`.

Reply:
Since it's a bit OOS, it's done in #2740 pointing to this branch.
Feel free to review now or later, we can merge it in this branch or merge it later in development if this branch gets merged first 😉